instrumentation/grails-3.0/javaagent/src/test/groovy/test/ErrorController.groovy

```diff
@@ -5,14 +5,16 @@
 
 package test
 
+import static io.opentelemetry.instrumentation.test.base.HttpServerTest.ServerEndpoint.ERROR
+
 import grails.artefact.Controller
 import grails.web.Action
 
 class ErrorController implements Controller {
 
   @Action
   def index() {
-    render "Error"
+    render ERROR.body
```

Comment:
I have no idea why closing the okhttp response seems to have correctly caused GrailsTest to fail - it is set up to map 500 through this controller, which is also why `errorPageSpans` is set in the test. I can't think of any reason for the number of spans to be correct but the body assertion to be wrong. /cc @laurit